database/setup/mongodb.conf

```diff
@@ -0,0 +1,5 @@
+sudo <texteditor> /etc/mongodb.conf
```

Comment:
Not a valid config file and since it doesn't contain the entire config it will most likely cause problems